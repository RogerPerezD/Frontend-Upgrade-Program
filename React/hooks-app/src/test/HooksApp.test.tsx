import { shallow } from "enzyme";
import { HooksApp } from "../HooksApp";

describe('Test sobre el componente HookApp', () => {
    
    test('should show the component succesfully', () => {
        
        const wrapper = shallow( <HooksApp/>);

        expect( wrapper ).toMatchSnapshot();
    });
});