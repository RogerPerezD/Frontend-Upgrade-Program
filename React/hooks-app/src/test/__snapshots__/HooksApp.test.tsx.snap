// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test sobre el componente HookApp should show the component succesfully 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
</Fragment>
`;
